identity/webapp/pages/index.tsx

```diff
@@ -241,7 +244,7 @@ const AccountPage: NextPage<Props> = ({ user: auth0UserClaims }) => {
         <>
           {!user?.emailValidated && (
             <AccountStatus type="info">
-              You have not yet validated your email address
```

Comment:
I binned the word 'validate' because it's not what we say in the emails or anywhere else.